src/index.js

```diff
@@ -0,0 +1,67 @@
+import './styles/tvmaze.css';
+import logo from './assets/tv-maze-logo.svg';
+import getData from './modules/data.js';
+import generateSingleShow from './modules/show.js';
+import { getLikes, addLike } from './modules/involvmentapi.js';
+
+const showItems = document.querySelector('.show-items');
+const tvMazeLogo = document.querySelector('.logo');
+const showCount = document.querySelector('.show-count');
+
+tvMazeLogo.src = logo;
+
+const renderLikes = (listItem) => {
+  getLikes().then((response) => {
+    response.forEach((e) => {
+      if (e.item_id === `${listItem.id}`) {
+        listItem.querySelector('.num-likes').innerText = e.likes;
+      }
+    });
```

Comment:
- [ ] To make your code more readable I would suggeste you to use `async` and `await ` instead of `then()`